### DISCUSSION TITLE
Meaningfulness of complex rules

### DISCUSSION THREAD

**Discussion by SichangHe** [SichangHe/internet_route_verification#185] [General]
> - Also, if "many of rules are simple" as shown in the text, how much of the verification result are complex and meaningful? Figs.2-4 only provided an overview of the potentially verified objects.

**Comment by cunha** [comment]
Section 2, after description of complex rule of AS14595, added:

> Such destination-specific traffic steering is an example of traffic engineering configurations that can only be captured precisely using compound RPSL rules.

**Comment by cunha** [comment]
The fraction of complex rules is given when we discuss compatibility with bgpq4; the Intro mentions that using complex rules requires more advanced tooling, which RPSLyzer helps with.
